Controller.php

```diff
@@ -157,8 +157,11 @@ private function renderManageContainerTemplate($template, $variables = array())
 
                 $mtmPreviewId = PreviewCookie::COOKIE_NAME . '=' .$idContainer;
                 $mtmPreviewId = SafeDecodeLabel::decodeLabelSafe($mtmPreviewId);
+                $previewCookie = new PreviewCookie();
+                $debugSiteUrl = $previewCookie->getDebugSiteUrl();
                 $disableLink = '<a class="title disablePreviewDebug" onclick=\'tagManagerHelper.disablePreviewMode(' . json_encode($release['idcontainer']) . ')\'>' . Piwik::translate('TagManager_DisablePreview') .'</a>';
-                $notification = new Notification(Piwik::translate('TagManager_PreviewDebugEnabledNotification', array($version, '<strong>', '</strong>', '<strong>?' . $mtmPreviewId . '</strong>', '<strong>&' . $mtmPreviewId . '</strong>', $disableLink)));
+                $urlInput = '<br><div class="preview-debug-url-div"><input type="url" id="previewDebugUrl" class="browser-default" placeholder="https://yourdebugsite.com" value="'.$debugSiteUrl.'"> <button type="button" class="btn btn-small" onclick=\'tagManagerHelper.changeDebugUrl(' . json_encode($release['idcontainer']) . ',"'.$debugSiteUrl.'")\'>Debug</button></div>';
```

Comment:
could we make the text field a bit wider @AltamashShaikh ? the URL is otherwise mostly hidden
![image](https://user-images.githubusercontent.com/273120/132621255-2227fce4-bfbc-4e13-a4f8-f16caebc11cb.png)

Reply:
Also can we move this part into a Twig file? You can then render this either using `renderTemplate()` or using `new View()...`;

The benefit is that you can escape things properly like the debug site url. In twig we can use different escaping strategies like `debugSiteUrl|e('html_attr')` and `debugSiteUrl|e('js')` etc to prevent XSS

Reply:
@mattab I'm assuming we would maybe want to prefill the URL with the configured main url (if any configured) ?
![image](https://user-images.githubusercontent.com/273120/132621740-2f46d174-e431-4d22-931f-00588728e39d.png)

Reply:
What would be the most useful from a user point of view? (what would 80% of users prefer to see in this field) @AltamashShaikh @tsteur

Reply:
I would say if only one domain is configured for a site, then we would prefill it with the main url, otherwise not prefill it. We would also prefill if they have configured the same domain twice for http and https like `http://www.example.com` and `https://www.example.com`. I probably wouldn't prefill if they configured `https://www.example.com` and `https://example.com`(this would be different subdomain). Most users will only have one domain and for these we could prefill? 
Other users can then still enter the URL (and path) if needed

Reply:
Made text field wider
Moved to twig

The URL is prefilled based on the current url we entered, for first time we get the site url using this below function
Site::getMainUrlFor($idSite);